### DISCUSSION TITLE
bin folder contents

### DISCUSSION THREAD

**Discussion by gaborschnierer** [Nice3point/RevitTemplates#46] [Q&A] [answered]
Hi Roman! Which method is generating these extra folders? 
![image](https://github.com/Nice3point/RevitTemplates/assets/30599750/0f14b4f0-42f5-4491-9117-95c6f2402eac)

**Comment by Nice3point** [comment, marked as answer]
Hi @gaborschnierer 
https://github.com/Nice3point/Revit.Build.Tasks#publishing
https://github.com/Nice3point/RevitTemplates/wiki/Third-party-files

Check these articles, this folder is used for publishing to the Revit addin folder, and for creating installer/bundle if you use solution template
Perhaps I should rename it `Revit 2025 bundle Release R25` to make it clearer or add subfolder `publish`

You can disable it, just add `<DisableAddinBundleGeneration>true</DisableAddinBundleGeneration>` in your csproj file

**Reply by Nice3point** [reply]
@gaborschnierer I changed the publish folder in version 1.1.1 and added a `publish` subfolder, just like it works with `dotnet publish`. Now the `bin` folder looks clearer and does not cause misunderstandings.
This fix is compatible with the current version of templates and existing projects.

If you are using a floating version of the [Revit.Build.Tasks](https://github.com/Nice3point/Revit.Build.Tasks) package, such as 1.*, the changes will be applied automatically when you open the IDE, if not, update the package to version 1.1.1

![изображение](https://github.com/Nice3point/RevitTemplates/assets/20504884/8dd0b1a3-a8ba-499c-8546-ce33c1844a51)
![изображение](https://github.com/Nice3point/RevitTemplates/assets/20504884/b22506c6-3855-4777-9395-98a6c551713e)

**Comment by gaborschnierer** [comment]
Thanks, this is exactly what I was looking for. I should've found these docs on my own. I guess I was overwhelmed. 😅
